opensource.html

```diff
@@ -1,28 +1,121 @@
+				
+				<p>Disagreements, social and technical, are normal, but we do not allow them to persist and fester leaving others uncertain of the agreed direction.</p>
+				
+				<p>We expect participants in the project to resolve disagreements constructively. When they cannot, we escalate the matter to structures with designated leaders to arbitrate and provide clarity and direction.</p>
+			</li>
+			<li>
+				<p>Ask for help when unsure</p>
+				
+				<p>Nobody is expected to be perfect in this community. Asking questions early avoids many problems later, so questions are encouraged, though they may be directed to the appropriate forum. Those who are asked should be responsive and helpful.</p>
+			</li>
+			<li>
+				<p>Step down considerately</p>
+				
+				<p>When somebody leaves or disengages from the project, we ask that they do so in a way that minimises disruption to the project. They should tell people they are leaving and take the proper steps to ensure that others can pick up where they left off.</p>
+			</li>
+		</ul>
+
+	<h4>Leadership, authority and responsibility</h4>
+	
+	<p>We all lead by example, in debate and in action. We encourage new participants to feel empowered to lead, to take action, and to experiment when they feel innovation could improve the project. Leadership can be exercised by anyone simply by taking action, there is no need to wait for recognition when the opportunity to lead presents itself.</p>
+
+	
+	<p>Open meritocracy</p>
+
+	<p>We invite anybody, from any company, to participate in any aspect of the project. Our community is open, and any responsibility can be carried by any contributor who demonstrates the required capacity and competence.</p>
+
+
+	<p>This Code is not exhaustive or complete. It is not a rulebook; it serves to distil our common understanding of a collaborative, shared environment and goals. We expect it to be followed in spirit as much as in the letter.</p>
+<h2>5.what is the practice for interacting with the community in terms of responding to issues/pull requests/comments?</h2>	
+<h3>Pull Requests</h3>
+	https://github.com/facebook/react/blob/master/CONTRIBUTING.md
+	The core team of repo will be monitoring for pull requests. From here, you'll need to get another person to sign off on the changes and then merge the pull request. 
+	<h4>Before submitting a pull request, please make sure the following is done…</h4>
+	<ul>
+		<li>Fork the repo and create your branch from master.</li>
+		<li>If you've added code that should be tested, add tests!</li>
+		<li>If you've changed APIs, update the documentation.</li>
+		<li>Ensure the test suite passes (relavant test suite).</li>
+		<li>If you haven't already, complete the CLA.(big corporations do this)</li>
+	</ul>
+<h2> how? </h2>
+<ul>
+	<li>Socialtables engineer could spent 3 hours a week working an open source project under socialtables repo</li>
+	<li>Projects could either benefit socialtables or other technologies</li>
```

Comment:
I think there should be more definition here. Maybe something like "Projects worked on for Social Tables impact Social Tables technologies or have a strong possibility of doing so." or something like that.